CHANGELOG.rst

```diff
@@ -2,6 +2,15 @@
 Changelog
 =========
 
+TBR
+------------------
+
+* Support for all existing capabilities at the moment
+* Introduction of ``_additional_attrs`` and ``AutoExtractAdapter``. Allows
```

Comment:
the name is different now

Reply:
Fixed here: https://github.com/scrapinghub/autoextract-poet/pull/14/commits/09705af76f4ffb2d7e44f3170bf9e2526abb3915